packages/component-library/package.json

```diff
@@ -32,12 +33,15 @@
   "dependencies": {
     "@types/classnames": "^2.2.6",
     "@types/lodash": "^4.14.132",
+    "@types/moment": "^2.13.0",
     "@types/react-select": "^3.0.5",
     "@types/uuid": "^3.4.4",
     "classnames": "^2.2.6",
     "lodash": "^4.17.11",
+    "moment": "^2.24.0",
     "motion-ui": "^2.0.3",
     "react-animate-height": "^2.0.15",
+    "react-calendar": "^2.19.2",
```

Comment:
Looks like you're not using `react-calendar` anywhere in here, this could be removed.